demisto_sdk/commands/format/update_generic.py

```diff
@@ -475,15 +476,25 @@ def adds_period_to_description(self):
         def _add_period(value: Optional[str]) -> Optional[str]:
             if value and isinstance(value, str):
                 strip_value = strip_description(value)
-                if not strip_value.endswith(".") and not is_string_ends_with_url(
-                    strip_value
+                if (
+                    not strip_value.endswith(".")
+                    and not is_string_ends_with_url(strip_value)
+                    and not is_sentence_ends_with_bracket(strip_value)
```

Comment:
just do `.endswith(")")`, no need for that method.

Reply:
since this method is used in the validation, I would like it to be as similar as possible,
[link](https://github.com/demisto/demisto-sdk/pull/3611) to `validate` pr.